src/docs/asciidoc/Sources.asciidoc

```diff
@@ -458,7 +458,7 @@ $$kafkaOffsetTopicName$$:: $$name of the offset topic, if Kafka offset strategy
 $$kafkaOffsetTopicRequiredAcks$$:: $$required acks for writing to the Kafka offset topic, if Kafka offset strategy is chosen$$ *($$int$$, default: `1`)*
 $$kafkaOffsetTopicRetentionTime$$:: $$retention time for dead records (tombstones), if Kafka offset strategy is chosen$$ *($$int$$, default: `60000`)*
 $$kafkaOffsetTopicSegmentSize$$:: $$segment size of the offset topic, if Kafka offset strategy is chosen$$ *($$int$$, default: `262144000`)*
-$$offsetStorage$$:: $$strategy for persisting offset values$$ *($$OffsetStorageStrategy$$, default: `kafka`, possible values: `inmemory,redis,kafka`)*
+$$offsetStorage$$:: $$strategy for persisting offset values$$ *($$OffsetStorageStrategy$$, default: `kafka_native`, possible values: `inmemory,redis,kafka,kafka_native`)*
```

Comment:
Good. Thanks.
So, what is the point of this noise here for review then?
Why can't it be generated during the `reference` task ?